llm/pdf-processing/requirements.txt

```diff
@@ -0,0 +1,3 @@
+unstructured[pdf,embed-huggingface]
```

Comment:
Just FYI - current version of `unstructured[pdf]` (`0.15.1`) does not work with the latest version of `nltk` which it uses for pdf processing. See https://github.com/iterative/datachain/pull/277 for some more details. Issue should be resolved in the next unstructured release.